src/Model/BookingRelay.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Date: 04-05-2020
+ * @author Raymond Kiekens
+ * @copyright (c) Oberon 2020
+ */
+
+namespace TOR\GraphQL\Model;
+
+class BookingRelay
```

Comment:
Deze titel is denk ik niet heel duidelijk, Relay is een facebook client library. Ik denk dat dit BookingConnection zou moeten heten (volgens mij zou de GraphQL type ook zo moeten heten).